src/enkf/gridio_gfs.f90

```diff
@@ -290,31 +291,124 @@ subroutine readgriddata_pnc(vars3d,vars2d,n3d,n2d,levels,ndim,ntimes, &
         end do
      end if
   endif
-  if (cw_ind > 0 .or. ql_ind > 0 .or. qi_ind > 0) then
-     call read_vardata(dset, 'clwmr', ug3d, ncstart=ncstart, nccount=nccount, errcode=iret)
-     if (iret /= 0) then
-        print *,'error reading clwmr'
-        call stop2(27)
+
+  ! ToDo: add handling for Thompson Scheme (imp_physcs: 8)
```

Comment:
This PR does not add support for imp_physics=8, right?  If true, we can remove this comment since it pertains to future work.